testing/workflows/components/unit_tests.jsonnet

```diff
@@ -106,6 +106,26 @@ local dagTemplates = [
     ]),  // create-pr-symlink
     dependencies: ["checkout"],
   },  // create-pr-symlink
+  {
+    template: buildTemplate("flake8-test", [
+      "python",
+      "-m",
+      "testing.test_flake8",
+      "--test_files_dirs=" +
+      srcDir + "/kubeflow/application/tests" + "," +
+      srcDir + "/kubeflow/common/tests" + "," +
```

Comment:
srcDir + "/testing" should also be added as it has most of the py

Reply:
DONE!